tests/bluetooth/iso/Kconfig

```diff
@@ -1,9 +1,14 @@
-config PRINT_CONN_INTERVAL
-	int "Print pr X connection interval"
-	default 100
+config PRINT_ISO_INTERVAL
+	int "Print pr X ISO interval"
```

Comment:
What is `pr X`?

Reply:
X is the value you set. So set 5 and it prints every 5th ISO interval.